insights/parsers/lvm.py

```diff
@@ -699,6 +699,9 @@ def _lvm_render(o):
     if isinstance(o, dict):
         parts = ['"%s": %s' % (k, _lvm_render(v)) for k, v in o.items()]
         return "{%s}" % ",".join(parts)
+    # for umask=077, it can not be parsed by json directly, transfer it to string first
+    if isinstance(o, str) and o.isdigit() and o.startswith('0') and len(o) > 1:
+        return '"%s"' % o
```

Comment:
From this page https://www.techoism.com/understanding-umask-value-in-linux/, I got 
```
The minimum and maximum UMASK value for a folder is 000 and 777
The minimum and maximum UMASK value for a file is 000 and 666
```

But when the value of the umask is not started with `0`, it won't cause issue #3475 and the value will be stored as `interge type` with the raw digit value. And it will result that for the same key `umask`, there would be different types of `int` and `str` for its values.

I think we should do more research and discussion as well.

Reply:
@xiangce How about we transfer the data before storing?